auth/realm/base/src/main/java/org/wildfly/security/auth/realm/FileSystemSecurityRealm.java

```diff
@@ -1035,7 +1121,26 @@ private void parsePublicKey(final List<Credential> credentials, final XMLStreamR
         private void parsePassword(final List<Credential> credentials, final XMLStreamReader streamReader) throws XMLStreamException, RealmUnavailableException {
             parseCredential(streamReader, (algorithm, format, text) -> {
                 try {
-                    if (BASE64_FORMAT.equals(format) || HEX.equals(format)) {
+                    if (ENCRYPTION_FORMAT.equals(format)) {
```

Comment:
Because `ENCRYPTION_FORMAT` was only introduced in version 1.1, we should have a check here that uses `version.isAtLeastVersion()` to make sure the version is >= 1.1. If not, an appropriate exception should be thrown.

Reply:
Added